homeassistant/components/zimi/__init__.py

```diff
@@ -0,0 +1,84 @@
+"""The zcc integration."""
+
+from __future__ import annotations
+
+import logging
+
+from zcc import ControlPoint, ControlPointError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.device_registry import format_mac
+
+from .const import DOMAIN, PLATFORMS
+from .helpers import async_connect_to_controller
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ZimiConfigEntry(ConfigEntry):
+    """Zimi ConfigEntry with correct runtime_data type."""
+
+    runtime_data: ControlPoint
```

Comment:
this can be simplified
```suggestion
type ZimiConfigEntry = ConfigEntry[ControlPoint]
```